Source/EasyNetQ/RabbitAdvancedBus.cs

```diff
@@ -106,30 +105,13 @@ IConsumeScopeProvider consumeScopeProvider
 
 
         /// <inheritdoc />
-        public bool IsConnected(PersistentConnectionType type)
-        {
-            return type switch
-            {
-                PersistentConnectionType.Producer => producerConnection.IsConnected,
-                PersistentConnectionType.Consumer => consumerConnection.IsConnected,
-                _ => throw new ArgumentOutOfRangeException(nameof(type), type, null)
-            };
-        }
+        public bool IsConnected => producerConnection.IsConnected && consumerConnection.IsConnected;
 
         /// <inheritdoc />
-        public void Connect(PersistentConnectionType type)
+        public void Connect()
         {
-            switch (type)
-            {
-                case PersistentConnectionType.Producer:
-                    producerConnection.Connect();
-                    break;
-                case PersistentConnectionType.Consumer:
-                    consumerConnection.Connect();
-                    break;
-                default:
-                    throw new ArgumentOutOfRangeException(nameof(type), type, null);
-            }
+            producerConnection.Connect();
```

Comment:
these feel like they should be async calls if possible

Reply:
But for these calls... I suggest moving to async only when we will to able to migrate both methods(Connect and CreateModel) to async. Is it fine?